src/browser-locale/index.js

```diff
@@ -0,0 +1,64 @@
+/* global TERRA_AGGREGATED_LOCALES */
+
+// The fallback locale that will be used if a supported locale is unable to be retrieved from the browser.
+const DEFAULT_LOCALE = 'en';
+
+/**
+ * Returns a boolean indicating if the locale is supported.
+ * @param {string} locale - The locale.
+ * @returns {bool} - A boolean indicating the locale is supported.
+ */
+const isSupported = (locale) => {
+  if (typeof TERRA_AGGREGATED_LOCALES === 'object' && Array.isArray(TERRA_AGGREGATED_LOCALES) && locale) {
+    return TERRA_AGGREGATED_LOCALES.indexOf(locale) > -1;
+  }
+
+  return false;
+};
+
+/**
+ * Filters the user's preferred languages from the browser and returns the first language found that is included in the supported locale list.
+ * @returns {string|null} - A supported locale. Null if not found.
+ */
+const filterLanguages = () => {
+  if (navigator.languages && navigator.languages.length > 0) {
```

Comment:
Looking at https://developer.mozilla.org/en-US/docs/Web/API/NavigatorLanguage/languages which was linked on the issue, it doesn't appear `navigator.languages` is not supported by IE 10. How do we plan to match this functionality in IE 10? Or how is this being handled?

Reply:
This is handled by the fallback strategy in the `getBrowserLocale`. It checks each available navigator accessor until a supported locale is found. If navigator.languages is undefined or does not contain a supported locale it will fallback to `language`, `userLanguage`, `browserLanguage`, and eventually `en` if no locale has been found.

Reply:
I see. I missed that logic flow with how this is broken out